code/DDSCodeTester.cpp

```diff
@@ -4626,6 +4626,11 @@ void dds_transport_examples ()
         // Link the Transport Layer to the Participant.
         qos.transport().user_transports.push_back(shm_transport);
         //!--
+ 
+        //CONF-SHM-TRANSPORT-DISABLE-BUILTIN-TRANSPORTS
+        // Explicit configuration of SharedMem transport
+        qos.transport().use_builtin_transports = false;
```

Comment:
This snippet is missing the creation of a custom SHM transport, and its addition to the participant. Same for the XML one.